backend/db-setup/reset-db.sql

```diff
@@ -11,4 +11,6 @@ GRANT SELECT, INSERT, DELETE, UPDATE, TRUNCATE ON TABLES TO simple_report_app;
 ALTER DEFAULT PRIVILEGES FOR USER simple_report_migrations IN SCHEMA simple_report
 GRANT SELECT, UPDATE ON SEQUENCES TO simple_report_app;
 GRANT USAGE ON LANGUAGE plpgsql to simple_report_app;
+DROP EXTENSION IF EXISTS pgcrypto;
+CREATE EXTENSION IF NOT EXISTS pgcrypto SCHEMA simple_report;
```

Comment:
This adds pgcrypto in the correct place for it to be used by migrations. The assumption was that the liquibase user might not have the requisite permissions to call `CREATE EXTENSION`, so we should make that a database management task like setting up the user and schema. It turns out that we have always configured our users in such a way that it could have been done in liquibase, but at this point it seems weird to go back and add it to the changelog as if we hadn't been doing it by hand all along.

Reply:
Oh, and the reason for the `DROP` is because if you add it globally rather than in the schema, it doesn't show up in the liquibase user's command path. 🤷